README.md

```diff
@@ -18,6 +18,26 @@ Navigate to the root of this repository and run `make fresh-env`
 
   #### a. For local development:
 
+  To install the necessary dependencies in the root directory run the commands:
```

Comment:
We already have the `make fresh-env` command to do this. We just need the npm install. So, please make changes inthe  Makefile to include it. There is no need to manually do it.